lms/djangoapps/discussion_api/views.py

```diff
@@ -144,6 +144,9 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
         * following: If true, retrieve only threads the requesting user is
             following
 
+        * view: "unread" for threads the requesting user has not read, or
```

Comment:
@mekkz Also, could you elaborate the 'refer to anything' part? A view is a filter  i.e. I want to "view" the unanswered posts.